assets/src/helpers/detectSwipe.ts

```diff
@@ -0,0 +1,79 @@
+import { SwipeDirection } from "../skate.d"
+
+const MIN_X = 30 // Min x swipe for horizontal swipe
+const MAX_X = 30 // Max x difference for vertical swipe
+const MIN_Y = 50 // Min y swipe for vertical swipe
+const MAX_Y = 60 // Max y difference for horizontal swipe
```

Comment:
Did you determine these numbers by testing on the tablet?

Reply:
No, they were cargo culted from a random example on the web. We can tweak them if we find a need.